### PR TITLE
[bitnami/redis-cluster] fix typo

### PR DESCRIPTION
**Pull request by mamil**
**Description of the change**
clean unnecessary blank and fix typo

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist**

- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by alemorcuq** [comment]
Thanks. Please, check the DCO for this commit.

**Comment by mamil** [comment]
@alemorcuq done.

File: bitnami/redis-cluster/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 annotations:
   category: Database
 apiVersion: v2
-appVersion: 6.2.6
+appVersion: 6.2.7
```
Comment (alemorcuq) [inline bitnami/redis-cluster/Chart.yaml:4]: You should increase `version`, not `appVersion`.

**Review by alemorcuq — CHANGES_REQUESTED** [review]
Thanks for the fix. Please, check my comment.

**Comment by mamil** [comment]
@alemorcuq thanks for your check, it's done again.

**Review by alemorcuq — APPROVED** [review]

**Pull request merged**